rollup-examples.config.js

```diff
@@ -1,12 +1,31 @@
+/**
+ * @author Garrett Johnson / http://gkjohnson.github.io/
+ */
+
 var path = require( 'path' );
 var fs = require( 'fs' );
 
+var srcFolder = __dirname + '/examples/jsm/';
+var dstFolder = __dirname + '/examples/js/';
+
+var files = [
+	// { path: /postprocessing[\/\\].+\.js$/ },
```

Comment:
No files are converted at the moment (so rollup will throw an error) but they can be added in future PRs. You can use rollup to convert the files in the jsm/controls folder by changing this to:

```js
var files = [
	{ path: /controls[\/\\].+\.js$/ }
];
```